utils.go

```diff
@@ -53,59 +49,3 @@ func SelectPeers(peers []*grpc.ClientConn, amount int) ([]*grpc.ClientConn, erro
 
 	return activePeers, nil
 }
-
-func ExportGraphDOT(round *Round, graph *Graph) {
```

Comment:
I feel we should still keep the following function in. Thoughts?

Reply:
I'll revert it.